src/amo/components/PromotedBadge/index.js

```diff
@@ -40,7 +40,7 @@ export const PromotedBadgeBase = ({
     case 'line':
       label = i18n.gettext('By Firefox');
       linkTitle = i18n.gettext(
-        'This is an official add-on built by the creators of Mozilla Firefox.',
+        'Official add-on built by Mozilla Firefox. Meets security and performance standards.',
```

Comment:
imo the phrasing is a little weird (e.g. "Meets security and performance standards." vs. "has been reviewed to meet our standards for security and performance") compared to the other two but 🤷 that's what's been decided.

Reply:
It is indeed what has been decided!